### PR TITLE
i2-FixItemUi

### PR DESCRIPTION
**Pull request by haynescrossley**
DRAFT

FixItemUI class ready for review before merging into master.

// to do - merge latest master

### CODE REVIEW

**Comment by haynescrossley** [comment]
@elliotmitchum FixItemUi class ready for review

**Comment by elliotmitchum** [comment]
@amrmosit This is ready for your review.

File: src/library/fixitem/FixItemUI.java [hunk 1]
```diff
@@ -4,97 +4,99 @@
...
+                String itemEntryString = getInput("Scan Item (<enter> completes): ");
+                if (itemEntryString.length() == 0) {
+                    control.processingCompleted();
+                }
+
                 else {
                     try {
-                        long itEM_Id = Long.valueOf(ITem_EnTrY_StRiNg).longValue();
-                        CoNtRoL.itemScanned(itEM_Id);
+                        long itEM_Id = Long.valueOf(itemEntryString).longValue();
```
Comment (amrmosit) [inline src/library/fixitem/FixItemUI.java]: PR 47 D1 CSG#2 All variable names are to start with lowercase letter and to be in camelBack

File: src/library/fixitem/FixItemUI.java [hunk 2]
```diff
@@ -4,97 +4,99 @@
...
                     catch (NumberFormatException e) {
-                        DiSpLaY_OuTpUt("Invalid itemId");
+                        displayOutput("Invalid itemId");
                     }
                 }
                 break;    
                 
             case INSPECTING:
-                String AnS = GeTiNpUt("Fix Item? (Y/N) : ");
+                String AnS = getInput("Fix Item? (Y/N) : ");
```
Comment (amrmosit) [inline src/library/fixitem/FixItemUI.java]: PR 47 D2 CSG#2 All variable names are meaningful and to start with lowercase letter and to be in camelBack

File: src/library/fixitem/FixItemUI.java [hunk 3]
```diff
@@ -4,97 +4,99 @@
...
                 boolean MuStFiX = false;
```
Comment (amrmosit) [inline src/library/fixitem/FixItemUI.java]: PR 47 D3 CSG#2 All variable names are to start with lowercase letter and to be in camelBack

**Review by amrmosit — CHANGES_REQUESTED** [review]
@haynescrossley @elliotmitchum There are 3 defects to fix

File: src/library/fixitem/FixItemUI.java (hunk 3 above)
Comment (haynescrossley) [reply src/library/fixitem/FixItemUI.java]: [7c0c240](https://github.com/elliotmitchum/ITC205/pull/47/commits/7c0c24052984db566dcc1c129970daac71f75de2)

File: src/library/fixitem/FixItemUI.java (hunk 2 above)
Comment (haynescrossley) [reply src/library/fixitem/FixItemUI.java]: [37b0aa2](https://github.com/elliotmitchum/ITC205/pull/47/commits/37b0aa2e4aaf64255e22546ee5536c04e9d5b179)

File: src/library/fixitem/FixItemUI.java (hunk 1 above)
Comment (haynescrossley) [reply src/library/fixitem/FixItemUI.java]: [2d2b5d8](https://github.com/elliotmitchum/ITC205/pull/47/commits/2d2b5d82544fa09692d3ac50aa7bf92cb98dc016)

**Comment by haynescrossley** [comment]
@amrmosit changes made - over to you for final review

**Review by amrmosit — APPROVED** [review]
@elliotmitchum 
Review is complete

**Comment by elliotmitchum** [comment]
Looks good to me, PR passes the build checks.

**Pull request merged**